example/webpack.config.js

```diff
@@ -54,8 +57,7 @@ if (production) {
     new webpack.DefinePlugin({
       'process.env': { NODE_ENV: JSON.stringify('production') }
     }),
-    new webpack.optimize.DedupePlugin(),
-    new webpack.optimize.OccurenceOrderPlugin()
```

Comment:
I think the configuration for `UglifyJsPlugin` is also not necessary any more since the options passed above for warnings and sourcemaps are the default in webpack 2, see [migration guide](https://webpack.js.org/guides/migrating/#uglifyjsplugin-sourcemap).